nautobot/dcim/models/power.py

```diff
@@ -137,31 +174,119 @@ class PowerFeed(PrimaryModel, PathEndpoint, CableTermination):
         "voltage",
         "amperage",
         "max_utilization",
+        "breaker_position",
```

Comment:
Does it actually make sense to clone `breaker_position` given it needs to be unique within a panel?